pkg/types/ovirt/defaults/machinepools.go

```diff
@@ -0,0 +1,42 @@
+package defaults
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+const (
+	//DAffinyGroupBase is a holder which will be replaced with the cluster name when creating the affinity group
+	DAffinyGroupBase                 = "CLUSTERNAME"
+	dControlPlaneAffinyGroupPostfix  = "-controlplane"
+	dControlPlaneAffinyGroupPriority = 5
+	dComputeAffinyGroupPostfix       = "-cluster"
+	dComputeAffinyGroupPriority      = 3
+)
+
+func setMachinePool(p *types.MachinePool) {
+	if p.Platform.Ovirt == nil {
+		p.Platform.Ovirt = &ovirt.MachinePool{}
+	}
+}
+
+// SetControlPlaneDefaults sets the defaults for the ControlPlane Machines.
+func SetControlPlaneDefaults(p *types.MachinePool) {
+	setMachinePool(p)
+	if p.Platform.Ovirt.AffinityGroups == nil {
+		p.Platform.Ovirt.AffinityGroups = []*ovirt.AffinityGroup{
+			{Name: DAffinyGroupBase + dControlPlaneAffinyGroupPostfix, Priority: dControlPlaneAffinyGroupPriority},
+			{Name: DAffinyGroupBase + dComputeAffinyGroupPostfix, Priority: dComputeAffinyGroupPriority},
+		}
+	}
+}
+
+// SetComputeDefaults sets the defaults for the Compute Machines.
+func SetComputeDefaults(p *types.MachinePool) {
+	setMachinePool(p)
+	if p.Platform.Ovirt.AffinityGroups == nil {
```

Comment:
```suggestion
	if len(p.Platform.Ovirt.AffinityGroups) == 0 {
```

Reply:
I'm ok with having 0 size, in that case no affinity groups will be created, that is the customization that will disable the affinity groups